File: Python/Pares.py
def main():
    num = int(input())
    array = [int(input()) for _ in range(num)]
    hay_pares = False

    for i in array:
        if i % 2 == 0:
            print(f"[{i}]", end=" ")
            hay_pares = True

    if not hay_pares:
        print(":(")


main()

File: C++/Pares.cpp
#include <iostream>
#include <vector>

using namespace std;

int main()
{
    int num;
    bool hayPares = false;

    vector<int> array(num);

    for (int i = 0; i < array.size(); i++)
    {
        cin >> array[i];
    }

    for (int numeros : array)
    {
        if (numeros % 2 == 0)
        {
            cout << "[" << numeros << "]";
            hayPares = true;
        }
    }

    if (!hayPares)
    {
        cout << ":(";
    }

    return 0;
}